Cargo.toml

```diff
@@ -7,10 +7,16 @@ license = "MIT/Apache-2.0"
 homepage = "https://github.com/rust-lang/time"
 repository = "https://github.com/rust-lang/time"
 documentation = "https://doc.rust-lang.org/time"
+keywords = ["no_std"]
 description = """
 Utilities for working with time-related functions in Rust.
 """
 
+[features]
+default = ["std"]
+std = []
+nightly = []
+
 [dependencies]
 libc = "0.2.1"
```

Comment:
Want to disable default features here so `libc` itself doesn't pull in the standard library?